web/themes/custom/usagov/usagov.theme

```diff
@@ -465,3 +466,68 @@ function usagov_theme_suggestions_alter(array &$suggestions, array $variables, s
     }
   }
 }
+
+function usagov_preprocess_breadcrumb(&$variables) {
+
+  $node = \Drupal::routeMatch()->getParameter('node');
+  if (empty($node)) {
+    return;
+  }
+
+  if (str_starts_with('/agencies/', $node->toUrl()->toString())) {
```

Comment:
Sorry but I got needle and haystack backwards on all of these, it should be the `$node->` part first

```php
if ($node->toUrl()->toString(), str_starts_with('/agencies/')) {
```